app/controllers/external_resources_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
   def update
     respond_to do |format|
       if @external_resource.update(external_resource_params)
-        format.html { redirect_to external_resources_path, notice: 'External resource was successfully updated.' }
+        format.html { redirect_to community_resources_path, notice: 'External resource was successfully updated.' }
```

Comment:
The PublicController endpoint is called community_resources, _and_, I think we should rename ExternalResource to CommunityResource in general. In that case we'd want to rename this route to something like `public_community_resources_path` ?